### DISCUSSION TITLE
GPT model with internet search for information

### DISCUSSION THREAD

**Discussion by FlyingHermes** [ChatGPTNextWeb/NextChat#3569] [Ideas]
We can add the ability to search for the desired information on the Internet using the module: https://github.com/polywrap/evo.ninja/tree/dev/packages/agents/src/agents/Researcher.

**Comment by lloydzhou** [comment]
v2.15.0支持插件：

https://github.com/ChatGPTNextWeb/NextChat-Awesome-Plugins/tree/main/plugins/arxivsearch
https://github.com/ChatGPTNextWeb/NextChat-Awesome-Plugins/tree/main/plugins/duckduckgolite
